src/pages/form-validation/password/index.tsx

```diff
@@ -0,0 +1,472 @@
+import React, { useState, useEffect, useCallback } from 'react';
+import {
+    AppBar,
+    Toolbar,
+    IconButton,
+    Typography,
+    TextField,
+    InputAdornment,
+    List,
+    ListItem,
+    InputProps,
+    Hidden,
+    Divider,
+    Button,
+    Card,
+} from '@material-ui/core';
+import MenuIcon from '@material-ui/icons/Menu';
+import { Done, Visibility, VisibilityOff } from '@material-ui/icons';
+import { makeStyles, useTheme, Theme } from '@material-ui/core/styles';
+import { useDispatch } from 'react-redux';
+import { TOGGLE_DRAWER } from '../../../redux/actions';
+import * as Colors from '@pxblue/colors';
+import { Spacer } from '@pxblue/react-components';
+import clsx from 'clsx';
+
+const useStyles = makeStyles((theme: Theme) => ({
+    containerWrapper: {
+        display: 'flex',
+        justifyContent: 'center',
+        alignItems: 'center',
+        flex: '1 1 0',
+    },
+    container: {
+        display: 'flex',
+        flexDirection: 'column',
+        flex: '1 1 0px',
+        alignItems: 'center',
+        backgroundColor: theme.palette.background.paper,
+        paddingTop: theme.spacing(4),
+        paddingLeft: theme.spacing(3),
+        paddingRight: theme.spacing(3),
+        maxWidth: 450,
+        maxHeight: 740,
```

Comment:
Instead of a magic number maxHeight, can we make the container height grow and shrink according to the content inside? Right now, between the Confirm Password text field and the divider, there is a big margin. Maybe we can limit this height to 32px tall?

![image](https://user-images.githubusercontent.com/8997218/133663845-a85b50bc-445a-4978-8df8-0710914fb22c.png)

Reply:
I've removed the magic height, the password example should grow and shrink based on the the amount of content displayed.